src/video/x11/SDL_x11modes.c

```diff
@@ -226,7 +226,7 @@ static SDL_bool SetXRandRModeInfo(Display *display, XRRScreenResources *res, RRC
             mode->refresh_rate = CalculateXRandRRefreshRate(info);
             ((SDL_DisplayModeData *)mode->driverdata)->xrandr_mode = modeID;
 #ifdef X11MODES_DEBUG
-            printf("XRandR mode %d: %dx%d@%dHz\n", (int)modeID,
+            printf("XRandR mode %d: %dx%d@%fHz\n", (int)modeID,
```

Comment:
This should be %gHz

Reply:
Applied in [9872d46](https://github.com/libsdl-org/SDL/pull/7368/commits/9872d4693a470edd625d6538331e8125c9fe2257)